modules/renter/contractor/contractor_test.go

```diff
@@ -186,6 +188,8 @@ func (stubHostDB) ActiveHosts() (hs []modules.HostDBEntry)
 func (stubHostDB) Host(types.SiaPublicKey) (h modules.HostDBEntry, ok bool)         { return }
 func (stubHostDB) PublicKey() (spk types.SiaPublicKey)                              { return }
 func (stubHostDB) RandomHosts(int, []types.SiaPublicKey) (hs []modules.HostDBEntry) { return }
+func (stubHostDB) IncrementSuccessfulInteractions(key types.SiaPublicKey)           {}
+func (stubHostDB) IncrementFailedInteractions(key types.SiaPublicKey)               {}
```

Comment:
oh.. didn't even notice I forgot the return. I will change that to match the style of the other declarations.